.circleci/config.yml

```diff
@@ -135,22 +135,6 @@ jobs:
       - *generate_base_path
       - run:
           command: npm run build
-      - *persist_to_workspace
```

Comment:
I also removed this because it was added along with the `upload_to_cloudfare`: https://github.com/GetJobber/atlantis/pull/1141/files#diff-78a8a19706dbd2a4425dd72bdab0502ed7a2cef16365ab7030a5a0588927bf47R138